Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/CachedQueries/FarePolicyInternal.hs

```diff
@@ -0,0 +1,224 @@
+{-
+ Copyright 2022-23, Juspay India Pvt Ltd
+
+ This program is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License
+
+ as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version. This program
+
+ is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+
+ or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details. You should have received a copy of
+
+ the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
+
+module Storage.CachedQueries.FarePolicyInternal where
```

Comment:
![Cropped_Image](https://github.com/nammayatri/nammayatri/assets/54761970/30241714-4d48-4acd-9a97-9e4f57cf8189)
for whole file.